src/ducks/client/utils.js

```diff
@@ -81,3 +81,8 @@ export const updateNotificationToken = (client, token) => {
 export const getNotificationToken = client => {
   return client.stackClient.oauthOptions.notification_device_token
 }
+
+export const fullyDehydrateDocument = doc => ({
```

Comment:
Why is it named « fully »? It doesn't seem we have a `dehydrateDocument` (or I missed it).

Reply:
Renamed to getDocumentIdentity, thanks I was not particularly happy with the name fullyDehydrateDocument since it can be confused with the dehydratation used for relationships.

Reply:
More explanation here : https://github.com/cozy/cozy-banks/pull/1711/commits/c104d08c8d29a3a8e0a04f8fe5fefc9942b9625d